src/10_functions.py

```diff
@@ -2,11 +2,19 @@
 
 # YOUR CODE HERE
 
+def is_even(num):
+  if num % 2 == 0:
```

Comment:
I believe there is no need to explicitly return True or False, You can just `return num % 2 == 0` which will return proper result with shorter syntax.

Reply:
Thanks for pointing that out 👍